Loop/Models/LoopError.swift

```diff
@@ -183,6 +190,9 @@ extension LoopError: LocalizedError {
         case .glucoseTooOld(let date):
             let minutes = formatter.string(from: -date.timeIntervalSinceNow) ?? ""
             return String(format: NSLocalizedString("Glucose data is %1$@ old", comment: "The error message when glucose data is too old to be used. (1: glucose data age in minutes)"), minutes)
+        case .glucoseInFuture(let date):
+            let minutes = formatter.string(from: -date.timeIntervalSinceNow) ?? ""
+            return String(format: NSLocalizedString("Glucose data is %1$@ in the future", comment: "The error message when glucose data is in the future to be used. (1: glucose data time in future in minutes)"), minutes)
```

Comment:
I think it should say that the future glucose data is invalid, and refer to the specific reading, rather than just "data": "Invalid glucose reading with a timestamp that is %1$@ in the future"

Reply:
The translation comment doesn't really make sense either.

Reply:
I fixed the first one but I'm not sure about where the translation comment is.

Reply:
The translation comment had a phrase "is in the future to be used", which I can't parse, but it looks like you fixed it. Thanks.